### DISCUSSION TITLE
Admin Notification

### DISCUSSION THREAD

**Discussion by ninjart** [awsmin/wp-job-openings#27] [Q&A] [answered]
Hi, 

I am wanting to send job applications to different emails based off another field filled by the application. Basically the ability to change the email address with a hook would be really helpful.

**Comment by anantajitjg** [comment, marked as answer]
For this, you can use the ```awsm_jobs_admin_notification_mail_headers``` hook and update the cc field for the email to send the notification.

https://github.com/awsmin/wp-job-openings/blob/0470a6c088343b5bfd568be4a6092c995bf15010/inc/class-awsm-job-openings-form.php#L750-L768

**Reply by ninjart** [reply]
Brilliant thanks, and is it possible to access the submitted form data in this method?

**Reply by anantajitjg** [reply]
Yes. The second parameter (```$applicant_details```) to the callback function will contain all the necessary details.
